README.md

```diff
@@ -155,12 +156,16 @@ file(s) | Comma delimited list of files you wish to process using edit-contacts.
 | documentID2            | false            |
 | documentID3            | true             |
 
-1. Use the following command to download and edit the documents:
+2. Use the following command to download and edit the documents:
```

Comment:
The syntax for ordered/numbered lists in markdown should be as this was, where the renderer will show them as 1,2,3 automatically so you don't have to go renumber them if you insert something in the middle later . That said, clearly it's not working:
![image](https://user-images.githubusercontent.com/1485206/132764268-0fd7dff7-36c4-4915-8c9f-e9c2189652ba.png)

given that, I don't know what formatting to change here to make it function as desired. Had you tried some stuff here already?

Reply:
I actually hadn't looked at alternatives, but your comment prompted me to. Indenting the content below the list items fixes the issue.

Reply:
Nice, thanks for taking care of that!